format/spec.md

```diff
@@ -44,6 +44,15 @@ The primary change in version 2 adds delete files to encode rows that are delete
 
 In addition to row-level deletes, version 2 makes some requirements stricter for writers. The full set of changes are listed in [Appendix E](#version-2).
 
+#### Version 3: Extended Types and Capabilities
+
+Version 3 of the Iceberg spec extends data types and existing metadata structures to add new capabilities, including:
+
+* Adding new data types, like nanosecond timestamps, variant, and unknown.
```

Comment:
I'm still not clear on scope here.  Should these be added incrementally as they are read (I think only nanosecond is finalized?)

Reply:
Yeah, I extracted this out of a larger set of changes. I can narrow this down.

Reply:
Removed the others that are missing. Those additions should update this.